addons/udes_stock/models/stock_move.py

```diff
@@ -156,11 +156,15 @@ def split_out_move_lines(self, move_lines):
                 'move_orig_ids': [],
                 'ordered_qty': total_ordered_qty,
                 'product_uom_qty': total_initial_qty,
-                'state': 'assigned',
+                'state': self.state,
+                # TODO: ^^^ Not always true if we split a move which
+                # is partially availabe
```

Comment:
This needs to be resolved.
Take a look at either `stock.move._get_relevant_state_among_moves()` or `stock.move._recompute_state()` from Odoo core stock module to see which is the most appropriate.